src/components/side_panel/pivot/pivot_layout_configurator/pivot_dimension/pivot_dimension.xml

```diff
@@ -20,7 +21,7 @@
           <t t-slot="upper-right-icons"/>
           <i
             class="o-button-icon fa fa-trash pe-1 ps-2"
-            t-if="props.onRemoved"
+            t-if="props.onRemoved and !isReadonly"
```

Comment:
Having to add an isReadonly condition on _every_ present and future button sounds like a nightmare and not somehting we want to do. You already forgot a button (hide/show measure), and to be complete we should be able to open the "Show measure as" side panel, but not to edit it.

Is it not possible to do something simpler, like a gray overlay above the panel that would stop all mouse events ?